articles/mongodb-vs-mysql/index.md

```diff
@@ -0,0 +1,79 @@
+| Organized By | Data is organized by a table, row, and columns. | Data is organized by collection, document, and field classification. |
+| Performance and Speed | It is better for smaller databases and more general solutions. | It is better for handling massive, unstructured data sets as it has in-memory speed. |
+| Website | Mysql.com | Mongodb.com |
+| Developer | Oracle corporation does the ongoing development. | MongoDB, Inc. does the ongoing development. |
+| Scaling | Allows using more powerful database servers (vertical scaling). | Allows adding more database servers (horizontal scaling). |
+| Support provider | It offers [Oracle lifetime support](https://www.mysql.com/support/), though at different versions. Each level offers a knowledge base, maintenance releases, bug fixes, patches, and updates. | Offers Enterprise-Grade Support. It allows flexibility to upgrade to the newer versions. It has unlimited access to security fixes, updates, and many more. |
+| Community of users | It has a large community of users, as it started several years ago. | It has a relatively smaller community than that of MySQL, as it is newer. |
+| Command speed | MySQL is slower than MongoDB in executing the same commands like select, update, and insert. | MongoDB is faster in the execution of commands than MySQL. |
+
+### MongoDB vs. MySQL: Advantages and disadvantages.
+
+#### Advantages of using MongoDB
+
+1. MongoDB's schema is flexible. It can be modified to fit the user's needs.
+2. It is scalable. Using replica sets in MongoDB is possible.
+3. It supports expansion in that it allows the addition of machines and RAM to the system. It is data-intensive, stores a lot of information, and queries a lot of data.
+4. It also allows for document validation and integrated systems.
+
+#### Disadvantages of using MongoDB
+
+1. It uses very high memory, and it may have massive data, some useless over time.
+2. It limits the document size to 16MB.
+3. In some instances, it is comparatively slow compared to NoSQL, which leads to poor page response.
+
+#### Advantages of using MySQL
+
+- It has been around for quite some time; hence publications and knowledge sources are many. It is a community-driven database.
+- It supports a password security system, and therefore it's somehow secure.
+- It can handle large unstructured data faster than MongoDB.
+
+#### Disadvantages of using MySQL.
+
+- It may take a lot of time and effort to perform some tasks, even for some tasks requiring an automatic update, such as incremental back-ups.
+- It is not scalable. It is not possible to use replica sets in MySQL.
+- It has no in-built [XML](https://en.wikipedia.org/wiki/XML#:~:text=Extensible%20Markup%20Language%20(XML)%20is,free%20open%20standards%E2%80%94define%20XML) (Extensible Markup Language) and [OLAP](https://olap.com/olap-definition/) (Online Analytical Processing).
+
```

Comment:
Can you take a practical point of view? Compare the two with respect to two projects that you may have worked on. Giving a real-world example helps the reader understand the pros and cons better. The pros and cons that you mention are good. But, I wouldn't be interested in them until I know their applications.